apps/root/src/constants/common.ts

```diff
@@ -66,93 +65,6 @@ export const DCA_PAIR_BLACKLIST = [
-  '0x1debd73e752beaf79865fd6446b0c970eae7732f', // Arbitrum - CBETH
-  '0x1dd5629903441b2dd0d03f76ec7673add920e765', // Polygon - jEUR
-  '0xcd0e5871c97c663d43c62b5049c123bb45bfe2cc', // Ethereum - USDC. Euler. Disabled due to hack.
-  '0xd4de9d2fc1607d1df63e1c95ecbfa8d7946f5457', // Ethereum - WETH. Euler. Disabled due to hack.
-  '0xc4113b7605d691e073c162809060b6c5ae402f1e', // Ethereum - DAI. Euler. Disabled due to hack.
-  '0x48e345cb84895eab4db4c44ff9b619ca0be671d9', // Ethereum - WBTC. Euler. Disabled due to hack.
-  '0xb95e6eee428902c234855990e18a632fa34407dc', // Ethereum - LUSD. Euler. Disabled due to hack.
-  '0x7c6d161b367ec0605260628c37b8dd778446256b', // Ethereum - wstETH. Euler. Disabled due to hack.
-  '0x1bbc5845e7e2000938c840f0343a975269c348ef', // Yearn - WBTC
-  '0x7597f9b2944809de4e5a841d6fa5e499099a5a42', // Arbitrum - MAI. Aave. Has less than 10k TVL so no defillama data.
-
-  '0xB562CC5300e2Dd51f9770923b52cE5F372f97a0e', // BNB Chain - Venus ADA - Hiding until release
-  '0xf3638ACAc71740e55C14d99e288F10ba4EDef348', // BNB Chain - Venus MATIC - Hiding until release
-  '0x0BADC4caa20E84e25e26a54CeC3faFeBcC7AB085', // BNB Chain - Venus DOGE - Hiding until release
-  '0xa2b783bB283cA621f32a8792FFE4A10ad817C039', // BNB Chain - Venus XRP - Hiding until release
-
-  '0x8214BC5875EF5A8A22a2d53F5E68c3Bb0b01B92e', // Optimism - Sonne MAI - Hiding until release
-
-  '0x67f70429750e38d6bae20839ace2692ff2eff52f', // Optimism - OP - Exactly
-  '0xe9a15d436390beb7d9eb312546de3fb01b1468ea', // Optimism - USDC - Exactly
-  '0x7eb7deac4929da076910c4aa35128ec70b0b820f', // Optimism - WETH - Exactly
-  '0xa61ef65f330bb9f3fce823aee704052d8dcdb27a', // Optimism - wstETH - Exactly
-
-  '0x57F93E3Fcb2b7eF126aeedA1f959c9EE4625aa70', // Optimism - MAI on Aave - Hiding because $MAI lost support on Aave
-  '0x7597f9b2944809de4e5a841d6fa5e499099a5a42', // Arbitrum - MAI on Aave - Hiding because $MAI lost support on Aave
-
-  '0xdac17f958d2ee523a2206206994597c13d831ec7', // Ethereum - USDT - Until we can figure out whats going on with approval
-  '0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56', // BSC - BUSD - Changing stablecoin to FDUSD
-
-  '0xd3594E879B358F430E20F82bea61e83562d49D48', // Optimism - PSP - Hiding until release
-
-  '0x83c0936d916d036f99234fa35de12988abd66a7f', // Polygon - GHST - Disable bc of aave vault full
-
-  '0x967Bd9D6aB1677Dd00DB8b4Fb85A41F1e979E49a', // Gnosis Chain - Agave USDC
-  '0xE6658DdF8A3b29E3043C1F24B823D7Abd529AD14', // Gnosis Chain - Agave USDT
-  '0x9197d01ee0dfdEe3c2af993207244612F9765A13', // Gnosis Chain - Agave WETH
-  '0xA00F5e7A17D2A626cE8eACEdD68CC146287262cB', // Gnosis Chain - Agave wstETH
-  '0x5d26F7e3F4556805d96F62528E0F1e39eD866029', // Gnosis Chain - Agave GNO
-  '0xF52Ad2D01bca6cfF1dd87386eE73b31f5b3d4B0f', // Gnosis Chain - Agave WXDAI
-  '0x4e3Decbb3645551B8A19f0eA1678079FCB33fB4c', // Polygon - JEur
-  ...SHAME_LIST,
-].map((a) => a.toLowerCase());
-
```

Comment:
🥹

Reply:
I think we might have to keep the shame list haha